### PR TITLE
fix scrapping for play by play and added deprecated team names

### PR DESCRIPTION
**Pull request by krlu**

### CODE REVIEW

File: basketball_reference_web_scraper/data.py [hunk 1]
```diff
@@ -44,12 +44,15 @@ class Team(Enum):
...
     # DEPRECATED TEAMS
+    KANSAS_CITY_KINGS = "KANSAS CITY KINGS"
     CHARLOTTE_BOBCATS = "CHARLOTTE BOBCATS"
     NEW_JERSEY_NETS = "NEW JERSEY NETS"
     NEW_ORLEANS_HORNETS = "NEW ORLEANS HORNETS"
     NEW_ORLEANS_OKLAHOMA_CITY_HORNETS = "NEW ORLEANS/OKLAHOMA CITY HORNETS"
     SEATTLE_SUPERSONICS = "SEATTLE SUPERSONICS"
+    ST_LOUIS_HAWKS = "ST. LOUIS HAWKS"
     VANCOUVER_GRIZZLIES = "VANCOUVER GRIZZLIES"
+    WASHINGTON_BULLETS = "WASHINGTON BULLETs"
```
Comment (jaebradley) [inline basketball_reference_web_scraper/data.py]: Minor: `WASHINGTON BULLETs` vs. `WASHINGTON BULLETS`

File: basketball_reference_web_scraper/html.py [hunk 2]
```diff
@@ -770,8 +771,10 @@ def team_names_query(self):
         return \
             '//*[@id="content"]' \
             '//div[@class="scorebox"]' \
-            '//div[@itemprop="performer"]' \
-            '//a[@itemprop="name"]'
+            '//strong' \
+            '//a'
+        # '//div[@itemprop="performer"]' \
```
Comment (jaebradley) [inline basketball_reference_web_scraper/html.py]: Feel free to remove these commented lines

**Review by jaebradley — COMMENTED** [review]
One minor typo to fix, otherwise, looks good!

**Comment by krlu** [comment]
@jaebradley I think I've addressed the comments above. 
Let me know if there's anything else I should change.

**Comment by jaebradley** [comment]
@krlu sorry about the delay - I've only recently returned from a vacation. This looks good - I'm going to merge. I plan on releasing in the next couple of days.

**Pull request merged**